src/components/shared/SingleUserPicker/SingleUserPicker.tsx

```diff
@@ -1,9 +1,8 @@
 import React, { ReactNode, useCallback } from 'react';
 import { defineMessages, MessageDescriptor, useIntl } from 'react-intl';
-import compose from 'recompose/compose';
 import classnames from 'classnames';
+import { useFormContext } from 'react-hook-form';
 
-import { useField } from 'formik';
 import { AnyUser } from '~data/index';
```

Comment:
`~data/index` doesn't exist in cdapp. Also, none of the message descriptors below have been updated to use new pattern.